.travis.yml

```diff
@@ -0,0 +1,7 @@
+
+language: node_js
+node_js:
+  - "8"
+  - "6"
+  - "node"
```

Comment:
Maybe should leave off "node" for Travis since it references the latest node and not necessarily the version we are using.
So it just takes longer to run tests and I'm not sure we get any value out of it.
If we decide we want to test, eg Node 9, we can use "9".

Reply:
I agree, great point. I will remove it.